fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/RankedSetHashFunctions.java

```diff
@@ -40,7 +40,8 @@ public class RankedSetHashFunctions {
     public static final String RANDOM = "RANDOM";
     public static final String MURMUR3 = "MURMUR3";
 
-    private static final RankedSet.HashFunction MURMUR3_HASH_FUNCTION = new GuavaHashFunction(Hashing.murmur3_32());
+    @SuppressWarnings("UnstableApiUsage")
+    private static final RankedSet.HashFunction MURMUR3_HASH_FUNCTION = new GuavaHashFunction(Hashing.murmur3_32_fixed());
```

Comment:
I figured I'd mention it here since I mentioned it with the other usage: here I'm pretty sure that there isn't a requirement for stability across releases. In fact, there's a "random" hash function implementation here that is unstable across executions, let alone releases, so this change here should be okay. (That's also, to my understanding, how the `RankedSetHashFunctions` are supposed to work.)